views/Admin/users.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends 'Admin/layout.html.twig' %}
+{% block content %}
+    <header class="jumbotron subhead" id="overview">
+        <h1><i class="glyph2x-group"></i>Users</h1>
+        <p class="lead pull-left">Manage Baïkal user accounts, and associated resources.</p>
+        <p class="lead pull-right"><a class="btn btn btn-inverse" href="#"><i class="icon-white icon-user"></i> + Add user</a></p>
+    </header>
+
+    <table class="table table-bordered table-striped users">
+        {% for user in users %}
+            <tr>
+                <td class="col-username">
+                    <i class="icon-user"></i> <strong>{{ user.username|escape }}</strong><br />
+                    {{ user.displayname|escape }} <a href="{{ user.mailtouri|escape }}">&lt;{{ user.email|escape }}&gt;</a>
+                </td>
+                <td class="col-actions no-border-left">
+                    <p class="pull-right">
+                        <nobr><a class="btn" href="{{ user.linkcalendars }}"><i class="{{ calendaricon }}"></i> Calendars</a></nobr>
```

Comment:
I just noticed that you just copied over the existing html verbatin. In that case: awesome! Keeps the PR smaller! We can overhaul the html later anyway ;)